src/pages/pegase/home/components/StudyTableDisplay.tsx

```diff
@@ -3,53 +3,80 @@
+  searchStudy: string | undefined;
+}
+
+interface UseStudyTableDisplayReturn {
+  rows: StudyDTO[];
+  lastPage: number;
+  page: number;
+  setPage: React.Dispatch<React.SetStateAction<number>>;
+}
+
+export const useStudyTableDisplay = ({ searchStudy }: UseStudyTableDisplayProps): UseStudyTableDisplayReturn => {
   const [rows, setRows] = useState<StudyDTO[]>([]);
+  const [lastPage, setLastPage] = useState(0);
+  const [page, setPage] = useState(1);
+
+  useEffect(() => {
+    setPage(1);
+    setLastPage(0);
+  }, [searchStudy]);
+
+  useEffect(() => {
+    fetch(BASE_URL + `/v1/study/search?page=${page}&size=${itemsPerPage}&search=${searchStudy}`)
+      .then((response) => response.json())
+      .then((json) => {
+        setRows(json.content);
+        setLastPage(Math.ceil(json.totalElements / itemsPerPage));
+      })
+      .catch((error) => console.error(error));
+  }, [page, searchStudy]);
+
+  return { rows, lastPage, page, setPage };
+};
+
+const StudyTableDisplay = ({ searchStudy }: StudyTableDisplayProps) => {
+  const { t } = useTranslation();
 
   const headers = [
-    columnHelper.accessor('id', { header: 'id' }),
-    columnHelper.accessor('study_name', { header: 'study_name' }),
+    columnHelper.accessor('study_name', { header: t('home.@study_name') }),
     columnHelper.accessor('user_name', {
-      header: 'user_name',
-      cell: ({ getValue }) => (
+      header: t('home.@user_name'),
+      cell: ({ getValue }: { getValue: () => string }) => (
         <StdAvatar size="s" backgroundColor="gray" fullname={getValue()} initials={getValue().substring(0, 2)} />
       ),
     }),
-    columnHelper.accessor('creation_date', {
-      header: 'creation_date',
-      cell: ({ getValue }) => <>{getValue().toLocaleDateString('fr-FR')}</>,
-    }),
-    columnHelper.accessor('keywords', {
-      header: 'keywords',
-      minSize: 500,
-      size: 500,
-      cell: ({ getValue, row }) => (
-        <div className="flex h-3 w-32">
-          <StdTagList id={`pegase-tags-${row.id}`} tags={getValue()} />
-        </div>
-      ),
-    }),
-    columnHelper.accessor('project', { header: 'project' }),
+    columnHelper.accessor('project', { header: t('home.@project') }),
+    columnHelper.accessor('status', { header: t('home.@status') }),
+    columnHelper.accessor('horizon', { header: t('home.@horizon') }),
+    columnHelper.accessor('keywords', { header: t('home.@keywords') }),
+    columnHelper.accessor('creation_date', { header: t('home.@creation_date') }),
   ];
+  const { rows, lastPage, page, setPage } = useStudyTableDisplay({ searchStudy });
 
-  useEffect(() => {
-    setTimeout(() => {
-      const studies = generateStudyRandomData(10 ** 2);
-      setRows(studies);
-    }, 1000);
-  }, []);
   return (
     <div className="h-60vh overflow-auto">
       <StdSimpleTable columns={headers} data={rows} />
+
+      <StdPagination lastPage={lastPage} currentPage={page} onChange={setPage} />
     </div>
```

Comment:
QCA : Proposition => 
```suggestion
    <>
      <div className="flex-1">
        <StdSimpleTable columns={headers} data={rows} />
      </div>
      <StdPagination lastPage={lastPage} currentPage={page} onChange={setPage} />
    </>
```